compiler_base/error/src/diagnostic/components.rs

```diff
@@ -351,7 +351,7 @@ impl Component<DiagnosticStyle> for CodeSnippet {
                     )) {
                     Some(sf) => {
                         for line in affected_lines.lines {
-                            let line_index = line.line_index.to_string();
+                            let line_index = (line.line_index + 1).to_string();
```

Comment:
Pls add more comments to explain why '+1' is here. In addition, it is necessary to consider that the documentation for users should indicate that compiler_base source map location meets the 1-based column specification.

Reply:
fixed